src/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -613,12 +613,16 @@ public static IEnumerable<object[]> Groups_CustomCulture_TestData()
         [MemberData(nameof(Groups_CustomCulture_TestData))]
         public void Groups(string pattern, string input, RegexOptions options, CultureInfo cultureInfo, string[] expectedGroups)
         {
-            CultureInfo originalCulture = null;
+            CultureInfo originalCulture = CultureInfo.CurrentCulture;
             try
             {
+                // In invariant culture, the unicode char matches differ from expected values provided.
+                if (originalCulture == CultureInfo.InvariantCulture)
+                {
+                    CultureInfo.CurrentCulture = new CultureInfo("en-US");
+                }
                 if (cultureInfo != null)
                 {
-                    originalCulture = CultureInfo.CurrentCulture;
                     CultureInfo.CurrentCulture = cultureInfo;
                 }
```

Comment:
got it. the change LGTM. thanks.

---
In reply to: [74858742](https://github.com/dotnet/corefx/pull/10806#discussion_r74858742) [](ancestors = 74858742)